src/BlockEntities/CommandBlockEntity.cpp

```diff
@@ -114,11 +113,10 @@ void cCommandBlockEntity::Activate(void)
 
 void cCommandBlockEntity::SetRedstonePower(bool a_IsPowered)
 {
-	if (a_IsPowered && !m_IsPowered)
+	if (a_IsPowered)
 	{
 		Activate();
 	}
-	m_IsPowered = a_IsPowered;
```

Comment:
This function has changed its contract. It used to mean "set the redstone power to either on or off, and if it is going from off to on, call Activate". Calling it to set "on" when the state was already "on" didn't call `Activate`. Now it calls `Activate` for each setting to "on". Is that intentional? If so, update the docs for the function everywhere.